packages/databricks-vscode/DATABRICKS.quickstart.md

```diff
@@ -10,95 +6,117 @@ The Databricks extension for VS Code allows you to develop for the Databricks La
 
 # Features
 
+-   Create [Databricks Asset Bundle](https://docs.databricks.com/en/dev-tools/bundles/index.html)
```

Comment:
Plural "Bundles" sounds better here, but maybe it's just me

Reply:
Yes, we need to always use "Databricks Asset Bundles" (plural), as in this case, or just "bundle" if singular is required (e.g., bundle configuration file, bundle root, etc.).